include/CppUTestExt/MockNamedValue.h

```diff
@@ -85,7 +85,6 @@ class MockNamedValue
     virtual void setObjectPointer(const SimpleString& type, const void* objectPtr);
     virtual void setSize(size_t size);
 
-    virtual void setComparator(MockNamedValueComparator* comparator);
```

Comment:
It allows to change the comparitor for a particular NamedValue. When removing this, the whole member comparitor_ would also be possible to be removed.

I can't argue I've used the setting of the comparitors in the MockNamedValue, but it feels like something that shouldn't be removed ;)